applications/clover_network.md

```diff
@@ -0,0 +1,182 @@
+
+#### Integration
+  
+  We decided to develop on Polkadot mainly because of its enhanced scalability and low-in-cost among other smart contract platforms in the space. As a sharded multichain network, Polkadot allows the processing of many transactions on different chains in parallel that most legacy networks do not. While providing scaling solutions through parachains, Polkadot development is steadily advanced, has gathered a large number of developers from the community, and numerous projects are emerging endlessly in the ecosystem. On the other hand, Polkadot’s Substrate technology provides a relatively easy-to-use and highly customizable framework that allows us to build our platform to communicate with other ledgers, which in a sense will likely attract more users to onboard in the future. 
+
+Clover will compete to join as parachain for Polkadot and is committed to becoming a digital finance portal and DeFi service provider on Polkadot. Focusing on both to-C and to-B users, Clover will provide a series of DeFi related products and services to meet the diverse needs of different users. At the same time, developing and providing a modular DeFi protocol, which greatly reduces the technology development threshold for upper-layer applications. Clover will put together a large user base and different projects into a one-stop open and integrated financial service platform on Polkadot, using Substrate framework, to communicate with other existing parachains such as Bitcoin and Ethereum parachains. Namely, Clover will be another parachain on Polkadot. Once the parachain gets the slot to join the Polkadot relay chain, they can communicate with each other through Cross-chain Message Passing (XCMP) protocol. That means Clover can build its DeFi with the other parachain assets, like the DeFi protocols on Ethereum.
+
+#### Team Interest
+
+  We are a team of like-minded people who have a dream of building a groundbreaking technology towards censorship resistance on utilizing newest additions to DLT technologies. Apart from incentivizing our core team with a structured/periodic compensation plan throughout the next 5 years with cliffs imposed in between, we give ourselves a high degree of autonomy and lots of space for creativity overall.
+
+### Project Details 
+
+#### Mockups/designs  
+[https://www.figma.com/file/SWg7LONt81ot05S1nlszcH/Clover-Wallet?node-id=0%3A1](https://www.figma.com/file/SWg7LONt81ot05S1nlszcH/Clover-Wallet?node-id=0%3A1)
+
+
+#### API specs
+[https://docs.google.com/document/d/1UfNWe82X76a1LoKrSwSXCJV9gbcbHT6QLtsU8DIYWzY/edit?usp=sharing](https://docs.google.com/document/d/1UfNWe82X76a1LoKrSwSXCJV9gbcbHT6QLtsU8DIYWzY/edit?usp=sharing)
```

Comment:
This appears to be a test run. Can you point to the actual API specs?

Reply:
Updated docs link accordingly: https://docs.google.com/spreadsheets/d/1BccgFYHS8YLEX5H9NpQf4ENE9Qwniq8YNpLEXXSAaRo/edit?usp=sharing